packages/angular_devkit/build_angular/src/angular-cli-files/utilities/bundle-calculator_spec.ts

```diff
@@ -126,7 +126,7 @@ describe('bundle-calculator', () => {
         {
           name: '0',
           original: {
-            filename: 'foo-es2015.js',
+            filename: 'foo-es2016.js',
```

Comment:
Oh uh. I didn't realize that we will start renaming the files in dist/ as well. Hmm. This file rename will break a lot of stuff in angular/angular (e.g  size tracking) and invalidate docs guides.

Let's back off on the es2016 change and keep only the modules change.

In v11 we can create a proper plan for target=es2016 or maybe even higher (zones!).

Reply:
I think we might want to name the files something more generic maybe instead of the syntax version.